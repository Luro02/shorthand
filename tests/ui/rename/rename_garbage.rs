use shorthand::ShortHand;

#[derive(ShortHand)]
struct Command {
    #[shorthand(rename(garbage = ""))]
    first: usize,
}

fn main() {}
