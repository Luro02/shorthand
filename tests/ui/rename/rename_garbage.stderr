error: unknown field `garbage`
 --> $DIR/rename_garbage.rs:5:24
  |
5 |     #[shorthand(rename(garbage = ""))]
  |                        ^^^^^^^
